CMakeLists.txt

```diff
@@ -129,3 +129,29 @@ function(trusted_firmware_build)
   target_link_libraries(tfm_api INTERFACE zephyr_interface veneer_lib)
 
 endfunction()
+
+if (CONFIG_TFM)
```

Comment:
How about the option config BUILD_WITH_TFM? Does this new option sort of overlap?

Reply:
Done